R/oauth.R

```diff
@@ -112,10 +112,31 @@ getGoogleTokenForSheet <- function(tokenFileId="", useCache=TRUE){
   secret <-  "wGVbD4fttv_shYreB3PXcjDY"
   appName = "google"
   endpointType = "google"
-  tokenFileName ="gs_token.rds"
-  scopeList <- c("https://spreadsheets.google.com/feeds","https://www.googleapis.com/auth/drive")
-  token = getOAuthToken(clientId, secret, appName, endpointType, scopeList, tokenFileName, tokenFileId, useCache)
-  token
+  # retrieve token info from environment
+  # main purpose is to enable server refresh
+  token_info <- getTokenInfo("googlesheets")
+  if(!is.null(token_info)){
+    HttrOAuthToken2.0$new(
+      authorize = "https://accounts.google.com/o/oauth2/auth",
+      access = "https://accounts.google.com/o/oauth2/token",
+      validate = "https://www.googleapis.com/oauth2/v1/tokeninfo",
+      revoke = "https://accounts.google.com/o/oauth2/revoke",
+      appname = appName,
+      key = clientId,
```

Comment:
please use obviously dummy clientId and secret here to avoid confusion.
Some comments would also help.